conductor_api/src/conductor/ui_admin.rs

```diff
@@ -349,6 +352,8 @@ id = 'test-bundle-id'"#,
                 port: 4000,
                 bundle: "test-bundle-id".into(),
                 dna_interface: None,
+                reroute_to_root: true,
+                bind_address: "127.0.0.1".to_string()
```

Comment:
I see the local host literal come up a few times. Is it worth it putting it in a static variable?